version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
I gave it a quick try and it is not trivial. Probably needs a custom release stage. Lets merge this as it is then.